GSA-TTS/FAC
# Sk/3955 blank finding refnums

**Closes**: https://github.com/GSA-TTS/FAC/issues/3955

**Testing**: 

1. Test with a failed audit: (dbkey = 70485, audit_year = 2020) if access to census data in Preview is available.

- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that dissemination_findingtext contains GSA_MIGRATION for blank reference_number.
- Verify that dissemination_finding contains GSA_MIGRATION for blank reference_number.
- Verify that dissemination_captext contains GSA_MIGRATION for blank reference_number.

2. Test by deleting FINDINGREFNUMS in an audit

- Pick an audit and delete FINDINGREFNUMS for a finding in census_historical_migration_elecauditfindings table.
- Run 'docker compose run --rm web python manage.py historic_data_migrator --years 'my_year' --dbkeys 'my_dbkey''
- Verify that dissemination_findingtext contains GSA_MIGRATION for blank reference_number.
- Verify that dissemination_finding contains GSA_MIGRATION for blank reference_number.
- Verify that dissemination_captext contains GSA_MIGRATION for blank reference_number.

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
